services/gitea/5.0.4/defaults/cm.yaml

```diff
@@ -52,3 +53,9 @@ data:
       readOnly: true
       mountPath: "/git-tls"
     clusterDomain: cluster.local.
+    memcached:
+      image:
+        tag: 1.6.15-debian-10-r0
+    postgresql:
+      image:
+        tag: 10.20.0-debian-10-r48
```

Comment:
do we want to stick with the same Postgres version if possible? 

I.e 11.11.0-debian-10-r62 which is Postgres 11 I think?

Reply:
Sounds good, thanks @cbuto !